borgmatic/actions/bootstrap.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import os
+import json
+
+import borgmatic.borg.extract
+import borgmatic.borg.rlist
+import borgmatic.config.validate
+import borgmatic.hooks.command
+
+from borgmatic.borg.state import DEFAULT_BORGMATIC_SOURCE_DIRECTORY
+
+logger = logging.getLogger(__name__)
+
+def get_config_paths(bootstrap_arguments, global_arguments, local_borg_version):
```

Comment:
As you do tests, it would be great if you could also flesh out these functions with docstrings. That includes basic info what arguments it accepts, what it returns, what it _does_, and stuff like "what exception types does this thing raise and when?" (Grep the codebase for "Raise" for examples.)